composer.json

```diff
@@ -22,6 +22,9 @@
         "drupal/jsonapi": "1.0.0",
         "drupal/simple_oauth": "2.0-rc2"
     },
+    "require-dev": {
+        "roave/security-advisories": "dev-master"
```

Comment:
Well
a) Their documentation uses require instead of require-dev
b) Require-dev is root only, so it would have to live in https://github.com/acquia/reservoir-project instead